api/app/controllers/spree/api/v2/storefront/account/orders_controller.rb

```diff
@@ -0,0 +1,75 @@
+module Spree
+  module Api
+    module V2
+      module Storefront
+        module Account
+          class OrdersController < ::Spree::Api::V2::BaseController
+            include Spree::Api::V2::CollectionOptionsHelpers
+            before_action :require_spree_current_user
+
+            def index
+              render_serialized_payload { serialize_collection(paginated_collection) }
+            end
+
+            def show
+              spree_authorize! :show, resource
+
+              render_serialized_payload { serialize_resource(resource) }
+            end
+
+            private
+
+            def paginated_collection
+              dependencies[:collection_paginator].new(collection, params).call
+            end
+
+            def collection
+              dependencies[:collection_finder].new.execute(user: spree_current_user)
+            end
+
+            def resource
+              scope.find_by(number: params[:id])
+            end
+
+            def scope
+              spree_current_user.orders.complete
```

Comment:
@przemosk let's add some includes to avoid n+1 queries:

```
def scope_includes
        {
          line_items: [
            variant: [
              :images,
              option_values: :option_type,
              product: :product_properties,
            ]
          ]
        }
      end
```